src/components/program-progress/ProgramProgressPage.jsx

```diff
@@ -23,12 +30,21 @@ const ProgramProgressPage = () => {
       if (!program) {
         return undefined;
       }
-
       return program.data;
     },
     [program],
   );
+  const courseData = program?.data?.courseData;
+  const totalCoursesInProgram = courseData?.notStarted?.length
+    + courseData?.completed?.length
+    + courseData?.inProgress?.length;
 
+  const coursesNotStarted = courseData?.notStarted;
+  const totalCoursesNotStarted = coursesNotStarted?.length;
+  let courseEndDate;
```

Comment:
There should be a default value set for the case when coursesNotStarted is an empty array